app/controllers/api/v1/users_controller.rb

```diff
@@ -10,6 +10,18 @@ class UsersController < ApplicationController
       # Returns: { data: Array[serializable objects] , errors: Array[String] }
       # Does: Creates and saves a new user record in the database with the provided parameters.
 
+      def index
+        users = User.all.map do |user|
+          {
+            id: user.id,
+            name: user.name,
+            email: user.email,
+            avatar: user_avatar(user)
+          }
+        end
+        render_json data: users, status: :ok
+      end
```

Comment:
Why would we ever need to return a list of all users?

Reply:
Administrators should be able to list/update all users

![image](https://user-images.githubusercontent.com/43917914/168654160-3c5d7e16-8ea8-4c76-9756-6474190adb1b.png)

Reply:
That's a completely random issue that has nothing to do with what you're working on